src/components/Button/Button.scss

```diff
@@ -151,6 +151,10 @@ $iconWidth: 16px;
             --yc-button-background-color: var(--g-color-base-brand);
             --yc-button-background-color-hover: var(--g-color-base-brand-hover);
 
+            &:focus::before {
+                outline-offset: 1px;
```

Comment:
This setting should be global for all components

Reply:
Decided to make changes only for action button